bin/reth/src/commands/stage/run.rs

```diff
@@ -146,15 +147,21 @@ impl Command {
         let mut provider_rw = factory.provider_rw()?;
 
         if let Some(listen_addr) = self.metrics {
+            let task_manager = TaskManager::current();
+            let task_executor = task_manager.executor();
```

Comment:
nit: stage command should simplu run with `ctx` and grab executor from there. would prefer to avoid instantiating task manager given it already exists but not accessible in current scope

Reply:
ah, good call, i wasn't aware we had a ctx with this already filled